uk.co.compucorp.civicrm.tasksassignments/scss/civitasks/modules/_civihr-ui-select.scss

```diff
@@ -41,6 +41,12 @@ $input-border-radius:    0;
     border-top: 0 !important;
   }
 
+  .select2-chosen {
+    span {
+      color: $field-data-color;
```

Comment:
`field-data-color` does not give any meaning or context , can you you more meaningful name that gives some more context to color and where its used?